app/src/main/java/openfoodfacts/github/scrachx/openfood/views/ContinuousScanActivity.java

```diff
@@ -143,7 +140,7 @@ public void barcodeResult(BarcodeResult result) {
             }
 
             lastText = result.getText();
-            if (!(isFinishing() || isDestroyed())) {
```

Comment:
why did we add isDestroyed ? Why are we removing it ?

Reply:
I remove it because it's not compatible api level 16
Added to avoid a bug when the barcodeResult method is called and activity is closed
I supposed isFinishing is doing the job